Makefile

```diff
@@ -29,6 +29,6 @@ local-test-e2e:
 	GO111MODULE=on \
 	operator-sdk test local ./test/e2e  \
 	--up-local \
-	--namespace operators \
+	--operator-namespace operators \
```

Comment:
hi, we need to add `--watch-namespace ""` as well. 
without that flag, `watch-namespace` will be same as `operator-namespace`

in fact, i made that chage in e2e test script. 
https://github.com/tektoncd/operator/blob/90100968fc4fc4788bcf4b1e6cc75344db349361/test/e2e-tests.sh#L26

but i set `--watch-namespace` to `tekton-pipelines`. this was a mistake. could you update that as well with (`--watch-namespace ""`
(the tests are passing because we are testing only in `tekton-piplines` namespace)